src/css/claiming.less

```diff
@@ -84,53 +84,58 @@ form {
   font-weight: normal;
 }
 
-[data-lang="es"] #claiming-social-security .hero_image  {
-  background-image: url('../images/retirement-hero_spanish.png');
+#claiming-social-security .intro {
+  margin-bottom: 30px;
+  .content-l_col + .content-l_col {
+    margin-top: 0;
+  }
 }
 
-[data-lang="en"] #claiming-social-security .hero_image  {
-  background-image: url('../images/retirement-hero.png');
+[data-lang="es"] #claiming-social-security .intro  {
+    margin-top: 30px;
 }
 
-#claiming-social-security .claiming-hero {
-  background-color: #FFCE8D;
+[data-lang="en"] #claiming-social-security .intro  {
+    margin-top: 15px;
 }
 
-#claiming-social-security .claiming-hero .hero_wrapper {
-  position: relative;
+@media (min-width: 850px) {
```

Comment:
There's a widow-orphan situation in the hero text for screen sizes from 850px to 891px.

![screen shot 2017-09-12 at 2 38 10 pm](https://user-images.githubusercontent.com/1183435/30342561-4a1b3436-97c8-11e7-949f-3c7f28a144a3.png)